website/src/css/index.scss

```diff
@@ -786,36 +786,3 @@ html[data-theme="dark"] .Section p a {
     grid-template-areas: "first" "last";
   }
 }
-
-/* Twitter Follow Button */
-
-.twitter-follow-button {
```

Comment:
Did you intend to remove also those other occurrences:
```
website/src/css/index.scss
311:  .twitter-follow-button,
672:  .twitter-follow-button {
792:.twitter-follow-button {
```

Reply:
Intentionally preserved — they are part of the custom site styling to position these buttons ✅